CONTRIBUTING.md

```diff
@@ -78,7 +78,13 @@ The test module, clusterctl, and experiments do not provide any backward compati
 #### Backporting a patch
 
 We only accept backports of critical bugs, security issues, or bugs without easy workarounds, any
-backport MUST not be breaking for either API or behavioral changes.
+backport MUST not be breaking for either API or behavioral changes. In order to improve user/developer experience
+maintainers can choose to backport:
+- Doc improvements
+- Improvements to CI signal
+- Improvements to the test framework (enabling improvements to provider's test signal)
+- Cert-manager bump (to avoid having branch using cert-manager versions out of support, when possible)
+
```

Comment:
We probably need something here about changes required to support patch versions of Kubernetes that are supposed to be supported by the minor release. These will mostly be in the domain of kubeadmcontrolplane, but might include changes to the etcd clients, coredns libraries etc...

Reply:
Good point (s/patch versions/minor versions/)